.pre-commit-config.yaml

```diff
@@ -494,19 +494,25 @@ repos:
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_mypy.sh"
         files: \.py$
-        exclude: ^dev|^provider_packages|^chart
+        exclude: ^dev|^provider_packages|^chart|^docs
       - id: mypy
         name: Run mypy for helm chart tests
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_mypy.sh"
         files: ^chart/.*\.py$
         require_serial: true
+      - id: mypy
+        name: Run mypy for /docs/ folder
+        language: system
+        entry: "./scripts/ci/pre_commit/pre_commit_mypy.sh"
+        files: ^docs/.*\.py$
+        require_serial: true
       - id: pylint
         name: Run pylint
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_pylint.sh"
         files: \.py$
-        exclude: ^scripts/.*\.py$|^dev|^provider_packages|^chart
+        exclude: ^scripts/.*\.py$|^dev|^provider_packages|^chart|^docs
```

Comment:
It can't find modules. and i have no idea how to get around it. The `/scripts/ ` and `/dev/` directory is also excluded for the same reason. (probably)